articles/portal/ptl-ref-release-notes.md

```diff
@@ -19,6 +19,20 @@ toc_mdlink: ptl-ref-release-notes.md
 
 This article provides a list of completed Portal releases.
 
+## Release 1.124.0
+
+Date: 30th January 2020
+
+Feature update
+
+Details:
+
+- Display 2FA seed on setup
+
+- Remove the 2FA page when logged in
+
+- Implement SSO login (feature flagged) 
```

Comment:
Feature flagged changes are not relevant to customers unless they are switched on. So in this instance I assume the "Implement SSO login" entry is not required.

Reply:
Removed

Reply:
For the other 2 entries can we create a sentence so when the customer reads this they understand what this means. If you look at the tickets, the wording is already there.
* Display plain text 2FA seed during setup process. Allowing customers that cannot use the QR code directly, to manually copy and paste this into desktop based password management apps.
* Security enhancement, removed the 2FA page when logged in.

Reply:
Changed

Reply:
Thanks